mne/stats/cluster_level.py

```diff
@@ -664,7 +664,7 @@ def _permutation_cluster_test(X, threshold, n_permutations, tail, stat_fun,
     """Aux Function.
 
     Note. X is required to be a list. Depending on the length of X
-    either a 1 sample t-test or an f-test / more sample permutation scheme
+    either a 1 sample t test or an F test / more sample permutation scheme
```

Comment:
For consistency with the rest of the docs, maybe we should say t-test and F-test (sorry about that, I didn't realize that before).